src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -797,6 +824,7 @@ using stan::model::index_min_max;
 using stan::model::index_multi;
 using stan::model::index_omni;
 using stan::model::nil_index_list;
+using stan::scalar_type_t;
```

Comment:
Please use the namespace when using it. We are trying to reduce this using lists rather than expand it :). I am fairly convinced now that this meta template stuff should live in stan::math, but that is a separate story.

Reply:
Yeah sure!

The meta template stuff is also being used now in the stan repo now def feel free to hit me up with any issues!

Reply:
I mostly just get confused about what is in math and what is not. scalar_type to me feels like a Math meta thing but idk. To me, the Stan repo should be just services and algorithms and making sure models compile.